R/support_panel.R

```diff
@@ -0,0 +1,207 @@
+library(shiny)
+library(shinyGovstyle)
+
+#' Support panel
+#'
+#' @description
+#' Create the standard DfE R-Shiny support and feedback dashboard panel.
+#'
+#' @param team_email Your team e-mail address, must be a education.gov.uk email
+#' @param repo_name The repository name as listed on GitHub
+#' @param ees_publication Whether the parent publication is hosted on Explore
+#' Education Statistics
+#' @param publication_name The parent publication name
+#' @param publication_slug The parent publication slug on Explore Education
+#' Statistics
+#' @param alt_href Alternative link to the parent publication (if not hosted on
+#' Explore Education Statistics)
+#' @param form_url URL to a feedback form for the dashboard
+#' @param cookie_status_output Name of cookie status output object, often
+#' "cookie_status"
+#'
```

Comment:
I think we've explicitly provided the packages in the function calls below, but probably best to get into the practice of defining required packages here using:
`#' import shiny shinyGovStyle`

Then when we run `devtools::document()`, that will add them to the NAMESPACE. This hasn't caused problems so far because all the dashboards already load shiny and shinyGovStyle anyway, but this would have caused problems for packages not already loaded by parent dashboards I think.